pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -427,8 +427,16 @@ func (f *framework) ApplyScoreWeights(pc *PluginContext, pod *v1.Pod, scores Plu
 			errCh.SendErrorWithCancel(err, cancel)
 			return
 		}
-		for i := range nodeScoreList {
-			nodeScoreList[i] = nodeScoreList[i] * weight
+
+		for i, score := range nodeScoreList {
+			// Skip score plugin if it returns invalid score.
+			if score > NodeScoreMax || score < NodeScoreMin {
+				klog.Errorf("Skip score plugin %q since it returns an invalid score %q, it should in the range of [0, 100] after normalizing.", pl.Name(), score)
```

Comment:
Could you please `NodeScoreMin` and `NodeScoreMax` instead of 0 and 100 in the error message?

Reply:
done